diff -ruN --exclude=CVS --exclude=.#* caudium-1.4/Makefile.in caudium-1.4-darwin/Makefile.in
--- caudium-1.4/Makefile.in	Fri Jun 18 12:11:34 2004
+++ caudium-1.4-darwin/Makefile.in	Wed Sep 15 08:21:26 2004
@@ -144,7 +144,7 @@
 # well
 #
 install_alt:
-	@$(SRCDIR)/mkdir -p $(DESTDIR)/etc/caudium$(EXTRAVER)/servers/
+	@$(SRCDIR)/mkdir -p $(DESTDIR)/$(prefix)/etc/caudium$(EXTRAVER)/
 	@$(SRCDIR)/mkdir -p $(DESTDIR)/$(prefix)/bin
 	@$(SRCDIR)/mkdir -p $(DESTDIR)/$(prefix)/lib/caudium$(EXTRAVER)
 	@$(SRCDIR)/mkdir -p $(DESTDIR)/$(prefix)/lib/caudium$(EXTRAVER)/lib/@PIKE_VERSION@
@@ -153,10 +153,9 @@
 	@$(SRCDIR)/mkdir -p $(DESTDIR)/$(prefix)/share/caudium$(EXTRAVER)/local_fonts
 	@$(SRCDIR)/mkdir -p $(DESTDIR)/$(prefix)/share/doc/caudium$(EXTRAVER)
 	@$(SRCDIR)/mkdir -p $(DESTDIR)/$(prefix)/share/man/man1
-	@$(SRCDIR)/mkdir -p $(DESTDIR)/var/cache/caudium$(EXTRAVER)
-	@$(SRCDIR)/mkdir -p $(DESTDIR)/var/log/caudium$(EXTRAVER)
-	@$(SRCDIR)/mkdir -p $(DESTDIR)/var/run/caudium$(EXTRAVER)
-	@$(SRCDIR)/mkdir -p $(DESTDIR)/usr/local/share/caudium$(EXTRAVER)/modules
+	@$(SRCDIR)/mkdir -p $(DESTDIR)/$(prefix)/var/cache/caudium$(EXTRAVER)
+	@$(SRCDIR)/mkdir -p $(DESTDIR)/$(prefix)/var/log/caudium$(EXTRAVER)
+	@$(SRCDIR)/mkdir -p $(DESTDIR)/$(prefix)/var/run/caudium$(EXTRAVER)
 	@for f in $(INSTALLFILES); do \
 	 echo "Copying $$f ..."; \
 	 cp -p $(SRCDIR)/$$f $(DESTDIR)/$(prefix)/share/doc/caudium$(EXTRAVER)/; \
@@ -170,7 +169,8 @@
 	 echo "Copying $$f ..."; \
 	 cp -pr $(SRCDIR)/server/$$f $(DESTDIR)/$(prefix)/lib/caudium$(EXTRAVER); \
 	done
-	@rm -rf `find $(DESTDIR)/$(prefix)/lib/caudium$(EXTRAVER) -name CVS -or -name start-caudium.in`
+	@rm -rf `find $(DESTDIR)/$(prefix)/lib/caudium$(EXTRAVER) -name CVS`
+	@rm -f `find $(DESTDIR)/$(prefix)/lib/caudium$(EXTRAVER) -name start-caudium.in`
 	@echo "Copying modules..."
 	@rm -rf `find $(DESTDIR)/$(prefix)/share/caudium$(EXTRAVER) -name CVS -print`
 	@for f in $(SRCDIR)/server/*; do \
@@ -179,7 +179,7 @@
 	 fi; \
 	done
 	@echo "Fixing paths..."
-	@MDIR="$(prefix)/lib/caudium$(EXTRAVER)"; CACHETOP="/var/cache/caudium$(EXTRAVER)"; \
+	@MDIR="$(prefix)/lib/caudium$(EXTRAVER)"; CACHETOP="$(prefix)/var/cache/caudium$(EXTRAVER)"; \
 	 for v in modules local_modules; do \
 	  MODS="\\\"$$MDIR/$$v/\\\",$$MODS"; \
 	 done; \
@@ -190,9 +190,9 @@
 	 fi; \
 	 sed -e "s;\(globvar(.*\"ModuleDirs\",\).*;\1({$$MODS}),;g" \
 	     -e "s;\(globvar(.*\"font_dirs\",\).*;\1({\"$(prefix)/share/caudium$(EXTRAVER)/local_fonts\",\"nfonts\"}),;g" \
-	     -e "s;\(globvar(.*\"logdirprefix\",.*\)\"../logs/\"\(.*\);\1\"/var/log/caudium$(EXTRAVER)/\"\2;g" \
+	     -e "s;\(globvar(.*\"logdirprefix\",.*\)\"../logs/\"\(.*\);\1\"$(prefix)/var/log/caudium$(EXTRAVER)/\"\2;g" \
 	     -e "s;\(globvar(.*\"cachedir\",.*\)\"/tmp/caudium_cache/\"\(.*\);\1\"$$CACHETOP/\"\2;g" \
-	     -e "s;\(globvar(.*\"pidfile\",.*\)\"/tmp/caudium_pid\"\(.*\);\1\"/var/run/caudium$(EXTRAVER)/caudium.pid\"\2;g" \
+	     -e "s;\(globvar(.*\"pidfile\",.*\)\"/tmp/caudium_pid\"\(.*\);\1\"$(prefix)/var/run/caudium$(EXTRAVER)/caudium.pid\"\2;g" \
 	  $(DESTDIR)/$(prefix)/lib/caudium$(EXTRAVER)/base_server/caudium.pike > \
 	  $(DESTDIR)/$(prefix)/lib/caudium$(EXTRAVER)/base_server/caudium.pike.new; \
 	 mv $(DESTDIR)/$(prefix)/lib/caudium$(EXTRAVER)/base_server/caudium.pike.new \
diff -ruN --exclude=CVS --exclude=.#* caudium-1.4/server/modules/auth/auth_userdb.pike caudium-1.4-darwin/server/modules/auth/auth_userdb.pike
--- caudium-1.4/server/modules/auth/auth_userdb.pike	Tue Aug 10 17:56:54 2004
+++ caudium-1.4-darwin/server/modules/auth/auth_userdb.pike	Wed Sep 15 08:21:26 2004
@@ -501,7 +501,7 @@
          method_is_not_shadow);
 
 #if constant(System.getpwent)
-  defvar("method", "file", "Password database request method",
+  defvar("method", "getpwent", "Password database request method",
          TYPE_STRING_LIST, 
          "What method to use to maintain the passwd database. "
          "'getpwent' is by far the slowest of the methods, but it "
diff -ruN --exclude=CVS --exclude=.#* caudium-1.4/server/modules/filesystems/userfs.pike caudium-1.4-darwin/server/modules/filesystems/userfs.pike
--- caudium-1.4/server/modules/filesystems/userfs.pike	Thu Aug  5 12:54:03 2004
+++ caudium-1.4-darwin/server/modules/filesystems/userfs.pike	Wed Sep 15 08:21:26 2004
@@ -290,7 +290,7 @@
 	 "was started as root.",
 	 0, uid_was_zero);
 
-  defvar("pdir", "public_html/", "Public directory",
+  defvar("pdir", "Sites/", "Public directory",
 	 TYPE_STRING,
          "This is the directory in the home directory of the users which "
 	 "contains the files that will be shown on the web. "
diff -ruN --exclude=CVS --exclude=.#* caudium-1.4/server/modules/scripting/cgi.pike caudium-1.4-darwin/server/modules/scripting/cgi.pike
--- caudium-1.4/server/modules/scripting/cgi.pike	Fri Oct  8 18:24:04 2004
+++ caudium-1.4-darwin/server/modules/scripting/cgi.pike	Wed Oct 27 17:40:14 2004
@@ -959,7 +959,7 @@
   defvar("chroot","", "Chroot path", TYPE_STRING|VAR_MORE,
          "This is the path that is chrooted to before running a program.");
 
-  defvar("searchpath", "NONE/", "Search path", TYPE_DIR,
+  defvar("searchpath", "/Library/WebServer/CGI-Executables/", "Search path", TYPE_DIR,
 	 "This is where the module will find the CGI scripts in the <b>real</b> "
 	 "file system.");
 
diff -ruN --exclude=CVS --exclude=.#* caudium-1.4/server/start-caudium.in caudium-1.4-darwin/server/start-caudium.in
--- caudium-1.4/server/start-caudium.in	Wed Oct 27 17:40:50 2004
+++ caudium-1.4-darwin/server/start-caudium.in	Wed Oct 27 17:40:10 2004
@@ -65,7 +65,8 @@
   ({"program", Getopt.HAS_ARG, ({"--program"})}),
   ({"version", Getopt.NO_ARG, ({"--version"})}),
   ({"piketrace", Getopt.MAY_HAVE_ARG, ({ "--piketrace", "-t" })}),
-  ({"help", Getopt.NO_ARG, ({"--help", "-?"})})
+  ({"help", Getopt.NO_ARG, ({"--help", "-?"})}),
+	({"caudiumctl", Getopt.NO_ARG, ({"--caudiumctl"})}),
 });
 
 // loader options (with defaults)
@@ -528,6 +529,15 @@
     write_help();
     return 1;
   }
+
+	// Don't let fink users shoot themselves in the foot and not use caudiumctl
+	if(search(argv, "--caudiumctl")==-1)
+	{
+		write("Please use the command:\n");
+		write("sudo @prefix@/bin/caudiumctl1.4 start\n");
+		write("to start Caudium\n");
+		return 1;
+	}
 
   parsed_args=parse_arguments(argv);
 
diff -ruN --exclude=CVS --exclude=.#* caudium-1.4/tools/caudium-rc_script.in caudium-1.4-darwin/tools/caudium-rc_script.in
--- caudium-1.4/tools/caudium-rc_script.in	Fri May 28 14:35:00 2004
+++ caudium-1.4-darwin/tools/caudium-rc_script.in	Thu Sep 16 19:28:42 2004
@@ -2,55 +2,34 @@
 #
 # $Id$
 #
-# init.d script for Caudium. Set the variables below to something fitting..
-# This is only an example script.
+# Startup/Shutdown script for Caudium. 
 #
-#############
+# DON'T CHANGE ANYTHING IN THIS SCRIPT, or these may be overriden at your next
+# Caudium upgrade.
+# You can change the settings in the file @prefix@/etc/default/caudium14.
 
+# Read the Caudium startup options config files
 #
-# This is actually for Debian, but might come useful
-#
-if [ -f /etc/default/caudium ]; then
-    . /etc/default/caudium
+if [ -f @prefix@/etc/default/caudium1.4 ]; then
+    . @prefix@/etc/default/caudium1.4
 fi
 
 # The server directory where the 'start' script is located.
 #
 caudiumhome=${caudiumhome:-@CAUDIUM_SERVERDIR@}
 
-# Set this to something unique to be able to stop, reload and restart
-# with this init script. It will override the setting in the config
-# interface. '0' is typically replaced with the uid.
-#
-pidfile=${pidfile:-/tmp/caudium_pid}
-
-# Set these to kill all processes owned by wwwuser on stop. Useful to
-# reap CGI scripts.
-#
+pidfile=${pidfile:-@prefix@/var/run/caudium1.4/caudium.pid}
 killallwww=${killallwww:-no}
 wwwuser=${wwwuser:-www}
 
 umask 022
 
-# If you want to start with another configuration directory:
-#
-# configdir=dirname
-
-# If you want to start with another log directory:
-#
-# logdir=dirname
-
-# Here you can add extra flags to the start script, like enabling or
-# disabling threads.
-# 
-# flags="--without-threads"
-# flags="--with-threads"
-
-### You should not _have_ to change anything below here...
-
 test -n "$pidfile" && flags="$flags --pid-file=$pidfile"
 test -n "$configdir" && flags="$flags --config-dir=$configdir"
 test -n "$logdir" && flags="$flags --log-dir=$logdir"
+
+# Tell the start script it is invoqued by caudiumctl
+flags="$flags --caudiumctl"
 
 case $1 in
 	'start')

diff -ruN --exclude=CVS --exclude=.#* caudium-1.4/server/etc/modules/CaudiumVersion.pmod caudium-1.4-darwin/server/etc/modules/CaudiumVersion.pmod
--- caudium-1.4/server/etc/modules/CaudiumVersion.pmod  Thu Oct 28 17:25:27 2004
+++ caudium-1.4-darwin/server/etc/modules/CaudiumVersion.pmod   Wed Nov  3 19:25:03 2004
@@ -25,7 +25,7 @@
 //! release
 constant __caudium_version__ = "1.4";
 constant __caudium_build__ = "5";
-constant __caudium_state_ver__ = "RC1";
+constant __caudium_state_ver__ = "RC2";
 
 //! any code may _append_ to this string - NEVER replace it!
 string __caudium_extra_ver__ = "";

