diff -urN caudium-20000817.orig/server/bin/pdbi.pike caudium-20000817/server/bin/pdbi.pike
--- caudium-20000817.orig/server/bin/pdbi.pike	Wed Aug  2 15:26:25 2000
+++ caudium-20000817/server/bin/pdbi.pike	Fri Aug 18 13:34:08 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/pike
+#!bin/pike
  * By using it you implicitly state that you are aware of the risks, and
  * that take upon yourself all the responsabilities for any damage,
  * direct or indirect including loss of profict from the use of this software.
diff -urN caudium-20000817.orig/server/bin/sqladduser.pike caudium-20000817/server/bin/sqladduser.pike
--- caudium-20000817.orig/server/bin/sqladduser.pike	Wed Aug  2 15:26:25 2000
+++ caudium-20000817/server/bin/sqladduser.pike	Fri Aug 18 13:34:37 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/pike
+#!bin/pike
 #!NO_MODULE
 /*
  * Caudium - An extensible World Wide Web server
diff -urN caudium-20000817.orig/server/bin/ultrasum.pike caudium-20000817/server/bin/ultrasum.pike
--- caudium-20000817.orig/server/bin/ultrasum.pike	Sat Aug 12 03:58:59 2000
+++ caudium-20000817/server/bin/ultrasum.pike	Fri Aug 18 13:35:07 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/pike
+#!bin/pike
 /*
  * Caudium - An extensible World Wide Web server
  * Copyright © 2000 The Caudium Group
diff -urN caudium-20000817.orig/server/protocols/tetris.pike caudium-20000817/server/protocols/tetris.pike
--- caudium-20000817.orig/server/protocols/tetris.pike	Sun Aug  6 20:50:44 2000
+++ caudium-20000817/server/protocols/tetris.pike	Fri Aug 18 13:35:24 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/pike
+#!bin/pike
 /*
  * Caudium - An extensible World Wide Web server
  * Copyright © 2000 The Caudium Group
diff -urN caudium-20000817.orig/src/cmods/UltraLog/bin/parseisocountry.pike caudium-20000817/src/cmods/UltraLog/bin/parseisocountry.pike
--- caudium-20000817.orig/src/cmods/UltraLog/bin/parseisocountry.pike	Mon Jul 31 16:32:15 2000
+++ caudium-20000817/src/cmods/UltraLog/bin/parseisocountry.pike	Fri Aug 18 13:35:40 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/pike
+#!bin/pike
 
 constant cvs_version = "$Id$";
 /* ftp://ftp.ripe.net/iso3166-countrycodes */
diff -urN caudium-20000817.orig/tools/addfp98.pike caudium-20000817/tools/addfp98.pike
--- caudium-20000817.orig/tools/addfp98.pike	Fri Jul 28 12:31:10 2000
+++ caudium-20000817/tools/addfp98.pike	Fri Aug 18 13:36:31 2000
@@ -1,3 +1,4 @@
+#!bin/pike
 /*
  * addfp98.pike
  *
@@ -10,7 +11,6 @@
  * $Id$
  */
 
-#!/usr/local/bin/pike
 #define FP_BASE "/usr/local/frontpage"
 
 import Getopt;
diff -urN caudium-20000817.orig/tools/compress.pike caudium-20000817/tools/compress.pike
--- caudium-20000817.orig/tools/compress.pike	Fri Jul 28 12:31:10 2000
+++ caudium-20000817/tools/compress.pike	Fri Aug 18 13:36:48 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/pike
+#!bin/pike
 
 object open(string f, string m)
 {
diff -urN caudium-20000817.orig/tools/convert_accessed.pike caudium-20000817/tools/convert_accessed.pike
--- caudium-20000817.orig/tools/convert_accessed.pike	Fri Jul 28 12:31:10 2000
+++ caudium-20000817/tools/convert_accessed.pike	Fri Aug 18 13:37:01 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/pike
+#!bin/pike
 
 /* $Id$
  */
diff -urN caudium-20000817.orig/tools/uncompress.pike caudium-20000817/tools/uncompress.pike
--- caudium-20000817.orig/tools/uncompress.pike	Fri Jul 28 12:31:10 2000
+++ caudium-20000817/tools/uncompress.pike	Fri Aug 18 13:37:14 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/pike
+#!bin/pike
 
 object open(string f, string m)
 {
