diff -U3 -r caudium-1.0.37.orig/tools/smartlink caudium-1.0.37/tools/smartlink
--- caudium-1.0.37.orig/tools/smartlink	Tue Feb 20 12:18:06 2001
+++ caudium-1.0.37/tools/smartlink	Tue Nov 20 17:12:57 2001
@@ -40,7 +40,6 @@
        tmp=`echo $1 | sed -e 's/^-R//g'`
      fi
      RPATH="$RPATH:$tmp"
-     RLIST="$RLIST $tmp"
     ;;
     -n32)
      SET_N32="-n32"
@@ -148,15 +147,9 @@
      LDOPTS="$LDOPTS -R$RPATH"
    ;;
    Linux\ 2.*)
-     if test x$LINKING != xno -a "x$RPATH" != x ; then
-      for a in $RLIST; do 
-       if test $LINKER_IS_LD = yes; then
-         LDOPTS="$LDOPTS -rpath $a"
-       else
-         LDOPTS="$LDOPTS -Wl,-rpath,$a"
-       fi
-      done 
-     fi
+     LD_LIBRARY_PATH="$RPATH"
+     LD_RUN_PATH="$RPATH"
+     export LD_LIBRARY_PATH LD_RUN_PATH
    ;;
    UnixWare\ 7.*)
      if test x$LINKING != xno -a "x$RPATH" != x ; then
@@ -178,8 +171,8 @@
 #  ;;
 #  *)
 #    if test "x$NO_ULIMIT" = "x"; then
-#      # Limit data segment size to 48MB.#
-      ulimit -d 49152 >/dev/null 2>&1
+#      # Limit data segment size to 48MB.
+#      ulimit -d 49152 >/dev/null 2>&1
 #    fi
 #  ;;
 #esac
diff -U3 -r caudium-1.0.37.orig/tools/smartlink.c caudium-1.0.37/tools/smartlink.c
--- caudium-1.0.37.orig/tools/smartlink.c	Tue Feb 20 12:18:06 2001
+++ caudium-1.0.37/tools/smartlink.c	Tue Nov 20 17:12:58 2001
@@ -1,5 +1,5 @@
 /*
- * $Id$
+ * $Id$
  *
  * smartlink - A smarter linker.
  * Based on the /bin/sh script smartlink 1.23.
@@ -87,7 +87,7 @@
 
   if (!strcmp(argv[1], "-v")) {
     fprintf(stdout,
-	    "$Id$\n"
+	    "$Id$\n"
 	    "Usage:\n"
 	    "\t%s binary [args]\n",
 	    argv[0]);
@@ -180,10 +180,10 @@
 	if (!argv[i][2]) {
 	  i++;
 	  if (i < argc) {
-	    rpath_in_use |= add_path(rpath, argv[i]);
+	    rpath_in_use = 0; /* |= add_path(rpath, argv[i]); */
 	  }
 	} else {
-	  rpath_in_use |= add_path(rpath, argv[i] + 2);
+	  rpath_in_use = 0; /* |= add_path(rpath, argv[i] + 2); */
 	}
 	continue;
       } else if ((argv[i][1] == 'n') && (argv[i][2] == '3') &&
@@ -210,7 +210,7 @@
     if(strchr(argv[1],' ')) len=strchr(argv[1],' ') - argv[1];
     if(len > 1 && argv[1][len-2]=='l' && argv[1][len-1]=='d')
     {
-      for(i=0; i<new_argc; i++) {
+      for(i=2; i<argc; i++) {
 	if (new_argv[i][0] == '-' && new_argv[i][1]=='W' &&
 	    new_argv[i][2]=='l' && new_argv[i][3]==',')
 	{
@@ -240,6 +240,7 @@
     new_argv[1] = "-n32";
   }
 
+#if 0
   if (ld_lib_path) {
     char *p;
 
@@ -251,6 +252,7 @@
     }
     rpath_in_use |= add_path(rpath, ld_lib_path);
   }
+#endif
 
   if (rpath_in_use) {
     /* Delete the terminating ':' */
